src/components/SingleTermView/OverView/Details.jsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Box,
+  Chip,
+  Grid,
+  Stack,
+  Typography
+} from "@mui/material";
+import { vars } from "../../../theme/variables";
+import React from "react";
+
+const { gray800, gray500 } = vars;
+const Details = () => {
+  const synonyms = [
```

Comment:
all the content term related should be a prop passed down to the component, 2 approaches suggested here:
- either we move all these data to prop that are passed down to the component, OR
- we pass to this component the term to be GET from the rest api and we fetch the data when mounting this component in a useEffect and we use useMemo to cache the data if the term did not change (this mostly for the multiple tabs in the single term view).